providers/implementations/encode_decode/decode_der2key.c

```diff
@@ -614,6 +636,169 @@ static void rsa_adjust(void *key, struct der2key_ctx_st *ctx)
 
 /* ---------------------------------------------------------------------- */
 
+#ifndef OPENSSL_NO_ML_DSA
+static void *
+ml_dsa_d2i_PKCS8(const uint8_t **der, long der_len, struct der2key_ctx_st *ctx)
+{
+    ML_DSA_KEY *key = NULL, *ret = NULL;
+    OSSL_LIB_CTX *libctx = PROV_LIBCTX_OF(ctx->provctx);
+    PKCS8_PRIV_KEY_INFO *p8inf = NULL;
+    const unsigned char *p;
+    const X509_ALGOR *alg = NULL;
+    int plen, ptype;
+
+    /*
+     * The private key format in PKCS8 is the 64-bytes (d, z) seed pair.
+     * Algorithm parameters must be absent.
+     */
```

Comment:
IIRC in ML-DSA the seed is actually 32 bytes `ξ` (xi) rather than `(d, z)`.

Reply:
removed